README.md

```diff
@@ -47,7 +47,7 @@ OPTIONS
   --start-date=start-date                       Only show log events on or after this date
 
 DESCRIPTION
-  Argg, this is only a subset of the log events and live tailing isn't quite ready! Think this is a killer feature? 
+  Argg, this is only a subset of the log events and live tailing isn't quite ready! Think this is a killer feature?
```

Comment:
This change can be reverted since it will be auto-generated on the next release.